tests/framework-tests/preferences/js/PanelsTests.js

```diff
@@ -1329,34 +1266,16 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         modules: [{
             name: "Test the text sizer settings panel",
             tests: [{
-                expect: 6,
+                expect: 1,
                 name: "Test the rendering of the text size panel",
                 sequence: [{
-                    func: "fluid.tests.switchWidgetType",
-                    args: ["{that}.options.testOptions.widgetType", "{textSizePanel}"]
-                }, {
-                    listener: "fluid.tests.testDefault",
```

Comment:
Shall the initial state of the text size panel be tested? The same comment applies to the line space panel.